### PR TITLE
Ds4 58 send start command to assembly station

### PR DESCRIPTION
**Pull request by Siole**
Merges DS52 into DS58 due to troubles with too many similarities

### CODE REVIEW

File: backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java [hunk 1]
```diff
@@ -1,31 +1,89 @@
...
+    @Autowired
     private AssemblyRepository assemblyRepository;
-    //private MqttClient mqttClient; FIXME: Implement after MQTT lesson
+    @Resource
+    private IMqttClient client;
     @Value("false")
     private boolean isHealthy;
 
+    @Resource
+    private static GetStatusResponse latestStatus;
```
Comment (3FCO) [inline backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java:29]: This shouldn't be static

File: backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java [hunk 2]
```diff
@@ -1,31 +1,89 @@
...
+    @Resource
+    private String[] message = {""};
```
Comment (3FCO) [inline backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java:31]: Java doesn't compile with these ``@Resources`` and therefore doesn't run
```suggestion
    private IMqttClient client;
    @Value("false")
    private boolean isHealthy;
    private GetStatusResponse latestStatus;
    private String[] message = {""};
```

**Review by 3FCO — CHANGES_REQUESTED** [review]

File: backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java (hunk 1 above)
Comment (Siole) [reply backend/Assembly/src/main/java/dk/sdu/controllab/assembly/service/AssemblyServiceImpl.java]: Agreed - thought I'd already fixed that - well it's fixed now :-)

**Review by 3FCO — APPROVED** [review]
Godt arbejde! 🫡

**Pull request merged**